.github/workflows/ci.yml

```diff
@@ -263,6 +263,8 @@ jobs:
             --format detailed
             --timeout 7
             --max-concurrency 6
```

Comment:
if we see this doesn't help as much as we hope for we might want to decrease this - too many concurrent requests to the same host might trigger some anti-DOS mechanisms

Reply:
Sure, let's see what those changes will bring us :D